src/index.html

```diff
@@ -17,6 +29,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="nav">
+        <div class="nav__logo">
+          <a
+            href="#"
+            class="logo__link"
```

Comment:
Consider adding a more descriptive `href` value instead of `#` to improve accessibility and provide meaningful navigation. If the links are placeholders, ensure to update them with actual URLs when available.